libsession/src/main/java/org/session/libsession/messaging/contacts/Contact.kt

```diff
@@ -36,21 +36,12 @@ class Contact(val sessionID: String) {
     /**
      * The name to display in the UI. For local use only.
      */
-    fun displayName(context: ContactContext): String? {
-        nickname?.let { return it }
-        return when (context) {
-            ContactContext.REGULAR -> name
-            ContactContext.OPEN_GROUP -> {
-                // In open groups, where it's more likely that multiple users have the same name,
-                // we display a bit of the Session ID after a user's display name for added context.
-                name?.let {
-                    return "$name (${sessionID.take(4)}...${sessionID.takeLast(4)})"
-                }
-                return null
-            }
-        }
+    fun displayName(context: ContactContext): String? = nickname ?: when (context) {
+        ContactContext.REGULAR -> name
+        // In open groups, where it's more likely that multiple users have the same name,
```

Comment:
"In communities, "...?

Reply:
it's called open groups in the line below. It'd be nice if we just called them one thing.

Reply:
Maybe change and refactor to `ContactContext.COMMUNITY`?

Reply:
there's 100+ matches of `opengroup` and 100+ of `open_group` in the codebase.

Additionally, it'd be nice to rename closed groups to groups, but they're both perhaps open for debate, and out of scope of this PR.

Reply:
Ticket for the open groups -> communities & closed groups -> groups refactor is: SES-2110